### DISCUSSION TITLE
Settings have disappeared?

### DISCUSSION THREAD

**Discussion by andihindle** [alexjustesen/speedtest-tracker#1306] [Q&A]
Hi — just updated from an older version (apologies, honestly don't remember the number) to 0.18.2..... it seems like the 'settings' option is missing entirely from the UI.  Is this just me?  Or a known issue?  Thanks!

**Comment by alexjustesen** [comment]
If you're updating from a super old version it's probably because your account isn't an `admin`. You can fix that by running the CLI command  `app:update-user-role`, [docs](https://docs.speedtest-tracker.dev/other/commands#application-commands).

**Reply by sobeservices** [reply]
Yah had this same issue but still not fixed

**Reply by alexjustesen** [reply]
You need to take action to fix it. Run the command above to fix the account roles.
